doc/dockerfile.rst

```diff
@@ -19,6 +19,12 @@ This guide will help you in preparing your Dockerfile so that it has the
 components needed to run JupyterHub, allowing it to work on Binder
 deployments.
 
+.. note::
```

Comment:
My attempt at making really sure people don't venture past this warning unless they really know what they are doing.

Reply:
Use important directive:
```
.. important::
```